app/views/partials/_navbar.html.erb

```diff
@@ -0,0 +1,21 @@
+<nav class="navbar navbar-expand-lg navbar-light bg-light" id='navbar'>
+  <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation">
+    <span class="navbar-toggler-icon"></span>
+  </button>
+  <div class="collapse navbar-collapse" id="navbarNav">
+    <ul class="navbar-nav">
+      <li class='nav-item active'>
+            <%= link_to 'Euphony', root_path, class: 'nav-link' %>
+      </li>
+      <li class="nav-item">
+        <a class="nav-link" href="#">Concert tickets</a>
```

Comment:
Try using this placeholder:
```
<%= link_to 'Concert tickets', '#', class: 'nav-link' %>
```
It's always better to use the helpers Rails gives us